enhancements/update/reduced_reboots.md

```diff
@@ -0,0 +1,328 @@
+---
+title: reduced-reboot-upgrades
+authors:
+  - "@sdodson"
+reviewers:
+  - @darkmuggle
+  - @rphillips
+  - @derekwaynecarr
+  - @crawford
+  - @dcbw
+  - @miabbott
+  - @mrunalp
+  - @zvonkok
+  - @pweil-
+  - @wking
+  - @vrutkovs
+approvers:
+  - @derekwaynecarr
+  - @crawford
+creation-date: 2020-01-21
+last-updated: 2020-01-21
+status: provisional
+see-also:
+  - https://github.com/openshift/enhancements/pull/585
+  - "/enhancements/eus-mvp.md"
+
+---
+
+# Reduced Reboot Upgrades
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement is intended to reduce host reboots when upgrading across two or
+more OpenShift minor versions by enabling an N-2 version skew policy between all
+host components and cluster scoped resources.
+
+## Motivation
+
+While OpenShift is designed to minimize workload disruption and risk associated
+with rolling reboots there exist a class of customers and workloads where
+reboots remain a disruptive and time consuming activity. Additionally, with the
+introduction of Extended Update Support (EUS) a new upgrade pattern will emerge
+where clusters run 4.6 for a year or more then rapidly upgrade across multiple
+minor versions in a short period of time. Those customers wish to complete their
+upgrades in a condensed time frame and with as few reboots as possible, they do
+not intend to run each minor version for an extended period of time.
+
+### Goals
+
+- Define testing requirements for N-2 host to cluster resource version skew
+- Define version skew policies for host and cluster scoped resources
+- Reduce reboots in accordance with our new tested policies
+
+### Non-Goals
+
+- Exceeding upstream's documented Kubelet version skew policies
+
+## Proposal
+
+### User Stories
+
+#### Node - Improve Upstream Kubelet Version Skew Testing
+
+Kubernetes defines a version skew policy(https://kubernetes.io/docs/setup/release/version-skew-policy/#kubelet)
+which allows for kubelet N-2 to be compatible with kube-apiserver version N. At
+this point in time OpenShift is not comfortable with the level of testing upstream
+and the intersection of the specific features of OpenShift. We should work to
+define and implement upstream testing changes which give us an appropriate level
+of confidence that N-2 version skew issues would be identified in the community
+whenever possible.
```

Comment:
Note that this is not currently verified in upstream CI -- I raised this for the [SIG Architecture agenda tomorrow](https://groups.google.com/g/kubernetes-sig-architecture/c/QX-4qq2krm0/m/998T3cJUBQAJ) to ensure that this has test coverage or to clarify the support policy.